code/processes/segmentedtickerplant.q

```diff
@@ -6,13 +6,29 @@
 // All - publish all data for table
 // Filtered - apply filters to published data, filters defined on client side
 
+chainedtp:@[value;`chainedtp;0b];  /- sets process up as a chained segmented tickerplant
+
+/ below variables only apply when process is set up as a Chained TP
+tickerplantname:@[value;`tickerplantname;`stp1];                /- tickerplant name to try and make a connection to  
+createlogs:@[value;`createlogs;1b];                             /- allow chained tickerplant to create a log file
+subscribeto:@[value;`subscribeto;`];                            /- list of tables to subscribe for
+subscribesyms:@[value;`subscribesyms;`];                        /- list of syms to subscription to
+replay:@[value;`replay;0b];                                     /- replay the tickerplant log file
+schema:@[value;`schema;1b];                                     /- retrieve schema from tickerplant
+
```

Comment:
Move this to a separate file in code/segmentedtickerplant and put into a namespace e.g. .sctp.tickerplantname